firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -463,4 +463,23 @@ public boolean didCrashOnPreviousExecution() {
   public void setCrashlyticsCollectionEnabled(boolean enabled) {
     core.setCrashlyticsCollectionEnabled(enabled);
   }
+
+  /**
+   * Enables/disables/clears automatic data collection config by Crashlytics.
+   *
+   * <p>If this is set, it overrides the data collection settings provided by the Android Manifest,
```

Comment:
Suggest replacing lines 470 and 471 with:

"* If not set, defaults to `True`. When set to `False`, disables automatic collection by overriding the data collections settings in the AndroidManifest.xml, as well as any Firebase-wide automatic data collection settings. When set to `null`, clears the Crashlytics data collection configuration (in this case, enabling data collection becomes dependent on other settings)."